cmd/strelaysrv/etc/linux-systemd/syncthing-relaysrv.service

```diff
@@ -3,10 +3,10 @@ Description=Syncthing relay server
 After=network.target
 
 [Service]
-User=syncthing-relaysrv
-Group=syncthing-relaysrv
-ExecStart=/usr/bin/relaysrv
-WorkingDirectory=/var/lib/syncthing-relaysrv
+User=syncthing-strelaysrv
```

Comment:
Maybe he things in here should be just "strelaysrv" now since st means syncthing? And the file name.